### PR TITLE
Wt 96 setup UI layer for greetings screen

### PR DESCRIPTION
**Pull request by k1mka**

### CODE REVIEW

File: lib/presentation/screens/greeting_screen/greeting_layout.dart [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import 'package:water_tracker/presentation/widgets/greeting_widget..dart';
+
+class GreetingLayout extends StatelessWidget {
+  const GreetingLayout({Key? key}) : super(key: key);
+
+  static const paddingHorizontal = EdgeInsets.symmetric(horizontal: 28.0);
+  static const largeSpace = SizedBox(height: 209);
```
Comment (zddarova) [inline lib/presentation/screens/greeting_screen/greeting_layout.dart]: do not like this. Mby just use center?

File: lib/presentation/widgets/greeting_widget..dart [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  static const heightWidget = 362.0;
+  static const widthWidget = 400;
+  static const titlePadding = EdgeInsets.symmetric(horizontal: 34);
+  static const titleDescription = EdgeInsets.symmetric(horizontal: 36);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: heightWidget,
+      width: 400,
```
Comment (zddarova) [inline lib/presentation/widgets/greeting_widget..dart]: Center?

**Review by zddarova — CHANGES_REQUESTED** [review]

File: lib/presentation/screens/greeting_screen/greeting_layout.dart [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  const GreetingLayout({Key? key}) : super(key: key);
+
+  static const paddingHorizontal = EdgeInsets.symmetric(horizontal: 28.0);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Padding(
+        padding: paddingHorizontal,
+        child: Column(
```
Comment (zddarova) [inline lib/presentation/screens/greeting_screen/greeting_layout.dart:19]: remove column

File: lib/presentation/widgets/greeting_widget..dart [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  static const widthWidget = 400;
+  static const titlePadding = EdgeInsets.symmetric(horizontal: 34);
+  static const titleDescription = EdgeInsets.symmetric(horizontal: 36);
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: SizedBox(
+        height: heightWidget,
+        width: 400,
```
Comment (zddarova) [inline lib/presentation/widgets/greeting_widget..dart]: could sized box be removed?

**Review by zddarova — CHANGES_REQUESTED** [review]

File: lib/presentation/widgets/greeting_widget..dart (hunk 4 above)
Comment (zddarova) [reply lib/presentation/widgets/greeting_widget..dart]: and rename file, it contains 2 dots

File: lib/presentation/screens/greeting_screen/greeting_layout.dart [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+            const Spacer(),
+            const GreetingWidget(),
+            const Spacer(),
+            CustomButton(
+              buttonColor: CustomTheme.buttonDarkColor,
+              textButtonColor: CustomTheme.decorationColor,
+              onPressed: () {
+                // TODO: CALL SOMETHING
+              },
+              text: 'Get started',
```
Comment (zddarova) [inline lib/presentation/screens/greeting_screen/greeting_layout.dart]: Move to locale

File: lib/presentation/screens/greeting_screen/greeting_layout.dart [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Padding(
+        padding: paddingHorizontal,
+        child: Column(
+          children: [
+            const Spacer(),
+            const GreetingWidget(),
+            const Spacer(),
```
Comment (zddarova) [inline lib/presentation/screens/greeting_screen/greeting_layout.dart:23]: Not sure about spacer, is it displaying correct?

Comment (zddarova) [reply lib/presentation/screens/greeting_screen/greeting_layout.dart:23]: Mby u want to call spaces fields?

File: lib/presentation/widgets/notification_widget.dart [hunk 7]
```diff
@@ -19,16 +19,26 @@ class NotificationWidget extends StatelessWidget {
...
       width: widthWidget,
-      decoration: const BoxDecoration(
+      decoration: BoxDecoration(
         color: CustomTheme.decorationColor,
-        borderRadius: BorderRadius.all(widgetRadius),
-        boxShadow: [CustomTheme.boxShadowTheme],
+        borderRadius: const BorderRadius.all(widgetRadius),
+        boxShadow: [
+          BoxShadow(
+            color: Colors.grey.withOpacity(0.1),
```
Comment (zddarova) [inline lib/presentation/widgets/notification_widget.dart]: Move to theme, it is really possible and not so hard

Comment (zddarova) [reply lib/presentation/widgets/notification_widget.dart]: Not with withOpacity constructor, but possible

**Review by zddarova — CHANGES_REQUESTED** [review]

File: lib/presentation/screens/greeting_screen/greeting_layout.dart (hunk 6 above)
Comment (k1mka) [reply lib/presentation/screens/greeting_screen/greeting_layout.dart:23]: It looks great, I want it

File: lib/presentation/widgets/notification_widget.dart (hunk 7 above)
Comment (k1mka) [reply lib/presentation/widgets/notification_widget.dart]: I need this constructor, I can't work with it

**Review by zddarova — APPROVED** [review]

**Pull request merged**